src/main/java/com/angelbroking/smartapi/utils/Utils.java

```diff
@@ -9,6 +9,10 @@ public static boolean isEmpty(final CharSequence cs) {
 		return cs == null || cs.length() == 0;
 	}
 
+	public static boolean isEmpty(final Number nm) {
```

Comment:
why number obj is needed?

Reply:
Number object is removed now, now it'll onlt check for Integer value object.